technical-reference/reference-technical-articles/automatic-income-allocation.md

```diff
@@ -0,0 +1,86 @@
+# Automatic income allocation
+
+The purpose of this document is to explain the technical details for distributing incomes generated by the Venus Protocol. The distributed incomes are generated from the spread of lending rates (borrow minus supply rates) and liquidations (part of the liquidation incentives). The distribution process for both the Core Pool and Isolated Pools as well as how the different amounts are allocated to different destinations based on specific rules and the approach for near-streaming income distribution are covered.```
+
+## Income Distribution Schema
+We want to distribute the protocol incomes this way *[Tokenomics](https://snapshot.org/#/venus-xvs.eth/proposal/0xc9d270ccecb7b91c75b95b8d9af24fc7c20cd38c0c0c44888ed4e7724f4e7ce9)*
+
+The Venus Protocol generates income from two main sources:
+
+- (A) Spread between the borrowing rate and the supply rate
+- (B) Part of the liquidation incentives
+
+The distribution of protocol incomes is as follows:
+
+**Schema 1:**
+
+- When: Income generated by (A) from markets supported by the Prime program.
+- Distribution:
+    - 40%: Risk fund
+    - 40%: Treasury
+    - 10%: XVS Vault rewards
+    - 10%: Venus Prime
+
+**Schema 2:**
+
+- When: Conditions for Schema 1 are not satisfied.
+- Distribution:
+    - 50%: Risk fund
+    - 40%: Treasury
+    - 10%: XVS Vault rewards
```

Comment:
```suggestion
Spread income from tokens not included in Venus Prime and liquidation incentive income are distributed to  the Risk fund, Treasury, XVS Vault rewards. Venus Prime token spread income is also distributed to the Prime program. Specific distribution percentages and details are included in the [Tokenomics](https://docs-v4.venus.io/governance/tokenomics).
```

Reply:
I will only add the link to Tokenomics, to avoid confusions and out of sync info